backend/metadata/databases/default/tables/public_Session.yaml

```diff
@@ -11,46 +11,53 @@ array_relationships:
     foreign_key_constraint_on:
       column: SessionId
       table:
-        name: AttendenceDocument
+        name: AttendanceDocument
```

Comment:
The relation's name still is `AttendenceDocuments`. It should be `AttendanceDocuments`.